pkg/neonvm/controllers/vm_controller.go

```diff
@@ -165,6 +161,17 @@ func (r *VMReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
+	// examine cpuScalingMode and set it to the default value if it is not set
+	if vm.Spec.CpuScalingMode == nil || *vm.Spec.CpuScalingMode == "" {
+		log.Info("Setting default CPU scaling mode", "default", r.Config.DefaultCPUScalingMode)
+		vm.Spec.CpuScalingMode = lo.ToPtr(vmv1.CpuScalingMode(r.Config.DefaultCPUScalingMode))
```

Comment:
question: Is the `vmv1.CpuScalingMode(...)` cast necessary here?

Reply:
dropped